src/plugin/src/main/java/com/defenseunicorns/uds/keycloak/plugin/RegistrationValidation.java

```diff
@@ -142,20 +119,6 @@ public void validate(final ValidationContext context) {
             errors.add(new FormMessage(RegistrationPage.FIELD_LAST_NAME, Messages.MISSING_LAST_NAME));
         }
 
-        // Require a DoD affiliation
-        if (Validation.isBlank(formData.getFirst("user.attributes.affiliation"))) {
-            errors.add(new FormMessage("user.attributes.affiliation", "Please specify your organization affiliation."));
-        }
-
-        // Require a rank
-        if (Validation.isBlank(formData.getFirst("user.attributes.rank"))) {
-            errors.add(new FormMessage("user.attributes.rank", "Please specify your rank or choose n/a."));
-        }
-
-        // Require an organization
-        if (Validation.isBlank(formData.getFirst("user.attributes.organization"))) {
-            errors.add(new FormMessage("user.attributes.organization", "Please specify your organization."));
-        }
```

Comment:
These were previously required, so this is change for these to be optional. Maybe this is okay, but I was hoping we could have these configurable (I guess making them configurable can be later work). Can we check with current users (Navy I guess?) if it's ok to make these optional

Reply:
By moving to the user managed attributes we can configure these fields to be required or not via the realm.json. Which is actually applicable to other fields in this RegistrationValidation.java class, the checks here are mostly redundant if only checking that the field is blank.